app/controllers/users/sessions_controller.rb

```diff
@@ -0,0 +1,3 @@
+class Users::SessionsController < Devise::SessionsController
+  after_action :verify_authorized, except: [:new, :create, :destroy]
```

Comment:
so just to clarify, on `:new` and `:create` we can't really authorize since we don't really know who the current user is (typically what you use to verify the action). Its possible we should authorize a destroy action to the session, to make sure its the right user destroying the session, but devise mostly handles this. This was the simplest thing that gets the current specs passing.